src/bot/types.ts

```diff
@@ -38,7 +38,7 @@ export type WebhookHandler<E extends WebhookEvents> = (
   ctx: Context,
   octokit: ExtendedOctokit,
   event: WebhookEventPayload<E>,
-) => Promise<PullRequestError | SkipEvent | FinishedEvent>;
+) => Promise<PullRequestError | SkipEvent | FinishedEvent | unknown>;
```

Comment:
This line troubles me a bit, but if typechecks are passing, we can get back to in later